analysis_options.yaml

```diff
@@ -6,5 +6,6 @@ analyzer:
   errors:
     omit_local_variable_types: ignore
     annotate_overrides: ignore
```

Comment:
Maybe we would be better off with an explicit `prefer_single_quotes: warning`?

Reply:
Would make sense, because we don't let prs with analyzer warnings go through anyways and it is a "wrong" spelling and not something that has been forgotten like type annationations which are added when it works.